src/locales/en/translation.json

```diff
@@ -147,7 +147,11 @@
       "explainVerificationDelay": "If you have just verified a contract, it should take a few minutes to update here."
     }
   },
-  "consensusSnapshot": "Consensus Snapshot",
+  "consensusSnapshot": {
+    "blockHeight": "Block Height <BlockHeight />",
+    "title": "Consensus Snapshot",
+    "totalCirculation": "This is <StakedPercentage /> of the total circulation"
```

Comment:
Saw this after looking at a screenshot `%` is missing.

Reply:
added formatter and a comment to validate this later when syncing with API